app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/noItemMessage/img/warning-inline.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="21" height="21" viewBox="0 0 21 21">
```

Comment:
Please store all static content like images and icons at `/src/common/img/`

Reply:
Done